src/sentry/models/organization.py

```diff
@@ -120,6 +122,53 @@ def get_for_user(self, user, scope=None, only_visible=True):
             return [r.organization for r in results if scope in r.get_scopes()]
         return [r.organization for r in results]
 
+    def get_member_top_dog_organizations(self, user_id: int, queryset: QuerySet = None) -> QuerySet:
+        from sentry.models import OrganizationMember, OrganizationMemberTeam, Team
```

Comment:
why import here?
circular reference? can we add a note why?

Reply:
no circular reference, moving to the top

Reply:
moving import to the top